django_scheduled_tasks/scheduler.py

```diff
@@ -49,5 +55,9 @@ def _load_tasks():
     soon = datetime.now() + timedelta(minutes=1)
 
     for task in tasks:
-        add_task(task.execute, task.interval_minutes, next_run_time=soon if task.onstart else None)
+        if task.interval_minutes:
+            add_task(task.execute, task.interval_minutes, next_run_time=soon if task.onstart else None)
+        else:
+            add_day_task(task.execute, task.day, task.hour, next_run_time=soon if task.onstart else None)
```

Comment:
I don't think a day task should run on start. I think the paradigm implies that it will only run at the specified day/time.